src/appleseed/renderer/kernel/lighting/backwardlightsampler.cpp

```diff
@@ -331,10 +354,30 @@ void BackwardLightSampler::collect_emitting_triangles(
                     emitting_triangle.m_triangle_support_plane = triangle_support_plane;
                     emitting_triangle.m_area = static_cast<float>(area);
                     emitting_triangle.m_rcp_area = static_cast<float>(rcp_area);
+                    emitting_triangle.m_triangle_prob = 0.0f;   // will be initialized once the emitting triangle CDF is built
                     emitting_triangle.m_material = material;
 
+                    const size_t emitting_triangle_index = m_emitting_triangles.size();
+                    
                     // Store the light-emitting triangle.
                     m_emitting_triangles.push_back(emitting_triangle);
+
+                    // Check if the emitting triangle will be sampled using CDF
+                    // and insert static probability into the cdf.
```

Comment:
More accurately:

    // When using the CDF, insert the light-emitting triangle into the CDF.